arangodb-net-standard/DocumentApi/IDocumentApiClient.cs

```diff
@@ -227,7 +240,8 @@ public interface IDocumentApiClient
         Task<PatchDocumentsResponse<U>> PatchDocumentsAsync<T, U>(
           string collectionName,
           IList<T> patches,
-          PatchDocumentsQuery query = null);
+          PatchDocumentsQuery query = null,
+          ApiClientSerializationOptions serializationOptions = null);
```

Comment:
This doesn't have an xml doc comment. Would be nice to add one so that we don't get the associated warnings. The one below as well.

Reply:
Thanks, Fixed.